contracts.go

```diff
@@ -224,3 +232,55 @@ func compileContract(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func reviseContract(cmd *cobra.Command, args []string) error {
+
+	// read the params file
+	params_b, err := ioutil.ReadFile("params.toml")
+	if err != nil {
+		return err
+	}
+
+	// read the contract template
+	template_b, err := ioutil.ReadFile("template.md")
+	if err != nil {
+		return err
+	}
+
+	// read history file
+	history_b, err := ioutil.ReadFile("history.toml")
+	if err != nil {
+		return err
+	}
+
+	// combine params, history, and template data
+	all_b := [3][]byte{params_b, template_b, history_b}
+	byte_array := make([]byte, 3, 3)
+
+	for _, element := range all_b {
+		byte_array = append(byte_array, element...)
+	}
```

Comment:
i'm pretty sure the for loop can be ommitted in favour of:
`byte_array = append(byte_array, all_b...)`
try running the `megacheck` tool to see what it says before making this change (https://github.com/dominikh/go-tools/tree/master/cmd/megacheck)